src/report/dto/create-report.dto.ts

```diff
@@ -11,6 +11,9 @@ export class CreateReportDto {
   @IsLatitude()
   latitude: number;
 
+  @IsNotEmpty()
+  type: string;
```

Comment:
I strongly advise converting this into a string enum, as outlined in the TypeScript documentation on string enums: https://www.typescriptlang.org/docs/handbook/enums.html#string-enums. Moreover, for enhanced validation, consider incorporating the ParseEnumPipe as described in the NestJS documentation on binding pipes: https://docs.nestjs.com/pipes#binding-pipes.